components/tooltip/README.md

```diff
@@ -67,7 +80,13 @@ The `d2l-tooltip` component is used to display additional information when users
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/inputs/input-text.js';
-  import '@brightspace-ui/core/components/tooltip/tooltip.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip.js';  <!-- docs: start hidden content --> window.addEventListener('load', function () {
```

Comment:
Note: starting `<!-- docs: start hidden content -->` and even actual code on the same line as the import is done intentionally.

In my local testing, I would find that doing it as 

```js
import '@brightspace-ui/core/components/tooltip/tooltip.js';  
<!-- docs: start hidden content -->  
window.addEventListener('load', function () {
```
would cause a newline (whitespace) to render on the page, making that look at bit awkward/unprofessional. Since this code is hidden anyway, I figured this is a reasonable workaround for still having any live demo code displayed in a nice way.

Reply:
Why is the code hidden in the first place? Is the code not necessary to reproduce the state that's being shown in the demo?

Reply:
This code isn't relevant to the demos that are shown, at least in my opinion. 

It's essentially there to allow for all the tooltips to automatically open when the consumer visits the Tooltip documentation page. However, normally, you have to hover/click/tab to the opener for the tooltip to show up (unless you specifically set it to `always-on`). So that code is essentially just some temp code to improve the documentation page experience for the consumer.

Ah, I see what you mean. A user might get confused when they see that, on Daylight, the tooltips start off opened, whereas when they use it, they can't reproduce that effect with our provided API.

I suppose another reason why it's hidden is because it would make the demo code that's displayed longer/noisier.

However, do you think we should still show it to the consumer?